src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/DockerProcessor.kt

```diff
@@ -30,14 +32,19 @@ class DockerProcessor {
 
             val containerConfig = ContainerConfig.builder()
                     .image("busybox")
-                    .cmd("sh", "-c", "md5sum", "<<<", quotedMessage)
+                    .cmd("sh", "-c", "echo $quotedMessage | md5sum")
                     .build()
 
             val creation = docker.createContainer(containerConfig)
             containerId = creation.id()
             docker.startContainer(containerId)
 
-            return docker.logs(containerId, DockerClient.LogsParam.stdout()).readFully()
+            var logs = ""
+            docker.logs(containerId, stdout(), stderr()).use({
```

Comment:
If you want to dive deeper into Kotlin magic, remove `logs` and `return` the result of 
```use({ stream -> stream.readFully() })```

Reply:
Done